app/src/androidTest/kotlin/org/wordpress/aztec/demo/tests/MixedTextFormattingTests.kt

```diff
@@ -1,18 +1,31 @@
 package org.wordpress.aztec.demo.tests
 
 import android.support.test.rule.ActivityTestRule
+import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
+import org.wordpress.aztec.AztecText
 import org.wordpress.aztec.demo.BaseTest
 import org.wordpress.aztec.demo.MainActivity
+import org.wordpress.aztec.demo.R
 import org.wordpress.aztec.demo.pages.EditorPage
+import org.wordpress.aztec.source.SourceViewEditText
 
 class MixedTextFormattingTests : BaseTest() {
 
     @Rule
     @JvmField
     var mActivityTestRule = ActivityTestRule(MainActivity::class.java)
 
+    @Before
+    fun init() {
+        val aztecText = mActivityTestRule.activity.findViewById<AztecText>(R.id.aztec)
+        val sourceText = mActivityTestRule.activity.findViewById<SourceViewEditText>(R.id.source)
+
+        aztecText.setCalypsoMode(true)
+        sourceText.setCalypsoMode(true)
```

Comment:
I'm fine with this, but would it be simpler to enable Calypso mode in `MainActivity` when `isRunningTest` is true (similar to how we decide whether to add demo content) instead of here before each test?

Reply:
Yeah, that would work, but not for all tests. Paragraph tests, for example, will fail unless Calypso mode is off. But since the majority of tests rely on it being enabled I'll just disable it for paragraphs.